env/templates/700-hook-ing.yaml

```diff
@@ -5,6 +5,11 @@ metadata:
   name: hook
   annotations:
     kubernetes.io/ingress.class: nginx
+{{- if eq .Values.certmanager.production "true" }}
+    certmanager.k8s.io/issuer: letsencrypt-prod
+{{- else }}
+    certmanager.k8s.io/issuer: letsencrypt-staging
+{{- end }}
```

Comment:
This if/else/end block is no longer required within the ingress

Reply:
It seems to be present on https://github.com/jenkins-x/environment-tekton-weasel-dev/blob/676ef0e92724ed89ae6193aa5a7ac20a2b01b1f8/env/templates/700-hook-ing.yaml#L8. 

Should be removed from all ingress resources? Can we use only `ertmanager.k8s.io/issuer: letsencrypt-prod`?

Reply:
Yes, should not be removed... I don't think the PR that James R raised on the boot config was merged before the weasel environment was created

Reply:
yeah that block can be removed as we use external dns and the new issuer format https://github.com/jenkins-x/environment-tekton-weasel-dev/blob/676ef0e92724ed89ae6193aa5a7ac20a2b01b1f8/env/templates/cert-manager-prod-issuer.yaml#L13-L26 this means annotations on ingress rules aren't needed.

Reply:
Should I remove it on all ingress resources?

Reply:
yes, if it's present 👍

Reply:
It was only on hook. All ingress should not have now any cert-manager annotations.